code/datums/religions.dm

```diff
@@ -0,0 +1,661 @@
+	H.equip_or_collect(new /obj/item/clothing/mask/surgical(H), slot_wear_mask)
+	H.equip_or_collect(new /obj/item/clothing/suit/storage/labcoat(H), slot_wear_suit)
+
+/datum/religion/busta
+	name = "Hardcore"
+	deity_name = "Bustatime"
+	bible_name = "The Hardcores"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Atmospheric Technician"
+	female_adept = "Atmospheric Technician"
+	keys = list("busta", "bustatime", "zas", "airflow", "hardcore", "hardcores")
+
+/datum/religion/busta/misc(var/mob/living/carbon/human/H)
+	if(!(M_HARDCORE in H.mutations))
+		H.mutations.Add(M_HARDCORE)
+	H.equip_or_collect(new /obj/item/clothing/shoes/magboots(H), slot_shoes)
+
+/datum/religion/self
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "God"
+	female_adept = "Goddess"
+	keys = list("me", "i", "myself", "narcissism", "self importance", "selfishness")
+
+/datum/religion/self/misc(var/mob/living/carbon/human/H)
+	name = "Cult of \the [H]"
+	bible_name = "A God Am I - The Teachings of \the [H]" //Quite literally
+	deity_name = "\The [H]" //Very literally, too
+
+/datum/religion/alcholol
+	name = "Cult of the Beer"
+	deity_name = "Hic"
+	bible_name = "The Drunken Ramblings"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Drunkard"
+	female_adept = "Drunkard"
+	keys = list("alcohol", "booze", "beer", "wine", "ethanol", "c2h6o")
+
+/datum/religion/robutness
+	name = "Robustness"
+	deity_name = "The Robust"
+	bible_name = "The Rules of Robustness"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Robuster"
+	female_adept = "Robuster"
+	keys = list("robust", "robustness", "strength")
+
+/datum/religion/suicide
+	name = "Thanatology" // Guess it works
+	deity_name = "The Grim Reaper"
+	bible_name = "The Sweet Release of Death"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Reaper"
+	female_adept = "Reaper"
+	keys = list("suicide", "death", "succumb")
+
+/datum/religion/communism
+	name = "Communism"
+	deity_name = "Karl Marx"
+	bible_name = "The Communist Manifesto"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Komrade"
+	female_adept = "Komrade"
+	keys = list("communism", "socialism")
+
+/datum/religion/communism/misc(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/head/russofurhat(H), slot_head)
+	H.equip_or_collect(new /obj/item/clothing/suit/russofurcoat(H), slot_wear_suit)
+
+/datum/religion/capitalism
+	name = "Capitalism"
+	deity_name = "Adam Smith"
+	bible_name = "The Wealth of Nations"
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Stockholder"
+	female_adept = "Stockholder"
+	keys = list("capitalism", "free market", "liberalism", "money")
+
+/datum/religion/capitalism/misc(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/head/that(H), slot_head)
+
+/datum/religion/america
+	name = "American Exceptionalism"
+	deity_name = "Goerge Washington"
```

Comment:
Not sure if typo or intentional.

Reply:
This was written like that in the original code

Reply:
It definitely was not.

https://github.com/vgstation-coders/vgstation13/blob/bd483ea93c1801b5fd45f38f3d5b8c01fa4cc7ad/code/game/jobs/job/civilian_chaplain.dm#L349